app/src/test/java/com/github/se/signify/model/home/quest/FirestoreQuestRepositoryTest.kt

```diff
@@ -82,7 +82,7 @@ class QuestRepositoryFirestoreTest {
 
       // Act & Assert
       var callbackTriggered = false
-      questRepositoryFirestore.init {
+      firestoreQuestRepositoryFirestore.init {
```

Comment:
```suggestion
      firestoreQuestRepository.init {
```

Reply:
Don't bother commenting the same mistake multiple times. It really bloats the review.